canonical-web-and-design/global-nav
# Remain positioned at the top of the page on small screens

## Done
The position of the global nav remains at the top on all screen sizes.
Fixed the chevron icon

## QA
- Pull this branch
- Run `./run`
- Open http://localhost:8300
- Check the global nav works as expected
- Shrink the screen to a small screen size
- Check the global nav stays at the top
- Check the product menu works as expected

Fixes https://github.com/canonical-web-and-design/global-nav/issues/108
